### ISSUE TITLE
BIMVie.ws

### ISSUE THREAD

**Issue by pakizito** [opensourceBIM/BIMserver#459]
Good afternoon,
Maybe I'm doing something wrong or I'm not sure what to expect regarding this:
After all the setup phase when installing the BIMServer I checkin a IFC file and I see the tree on BIMVie.ws, but when I click the 3D on the right side, I get a blank screen, is it supposed to be a BIMSurfer view of the project?
This is my current URL:
http://localhost:8080/bimserverwar-1.5.64/apps/bimviews
If I go to bimsurfer I get a 404:
http://localhost:8080/bimserverwar-1.5.64/apps/bimsurfer/
I see a little eye on the tree on each node and I click it and nothing happens...

I'm trying to figure out what could be the problem, and nothing until now...
Would this be something that people already experienced?
Thank you very much.

With kind regards,
Rodrigo Carvalheira

**Comment by raindog** [comment]
Same here... spent the day trying to figure out what i've done wrong... givin'up for today.

**Comment by rubendel** [comment]
This was not a BIMvie.ws problem, there was a bug in IfcPlugins. https://github.com/opensourceBIM/BIMserver/releases/tag/parent-1.5.65 is working here.

**Comment by raindog** [comment]
Ok, it works now, thank you Ruben.

**Comment by alanfoxye** [comment]
hi there. I got the same result using bimserverjar-1.5.67.jar. I downloaded Schependomlaan project after setup the BIMServer.I got the treeviews all right, but saw blank 3D view. And when I use BIMsurfer demo, I can see the projects list on BIMServer, but still can not get 3D view.
Using chrome I got these: 

[WARN] [xeogl.Canvas 2]: Failed to get a WebGL 2 context - defaulting to WebGL 1.
warn @ xeogl.js?bust=1487651109000:11102
xeogl.js?bust=1487651109000:11102 [WARN] [xeogl.Input 4]: Browser event not supported: orientationchange
warn @ xeogl.js?bust=1487651109000:11102
Notifier.js?bust=1487651109000:21 success Arguments[2]
Notifier.js?bust=1487651109000:25 info Arguments[2]
Notifier.js?bust=1487651109000:21 success Arguments[1]
BimSurfer.js?bust=1487651109000:234 Started loading geometries

then it got stuck on loading geometries......
